### PR TITLE
Rebrand: rename lazyledger to celestia

### PR DESCRIPTION
**Pull request by tzdybal**
- [x] fix links in readme
- [x] update `go.mod` and `go.sum`
- [x] fix import paths
- [x] use celestia-app
- [x] use plain Tendermint, not fork
- [x] update dependencies

### CODE REVIEW

**Comment by evan-forbes** [comment]
we should be able to finish this now that we have celestia-app and our fork of the sdk using celestia-core

**Comment by tzdybal** [comment]
> we should be able to finish this now that we have celestia-app and our fork of the sdk using celestia-core

That's great! Thanks for notification!

File: mempool/reactor.go [hunk 1]
```diff
@@ -0,0 +1,317 @@
...
+
+// Reactor handles mempool tx broadcasting amongst peers.
+// It maintains a map from peer ID to counter, to prevent gossiping txs to the
+// peers you received it from.
+type Reactor struct {
+	p2p.BaseReactor
+	config  *cfg.MempoolConfig
+	mempool *CListMempool
+	ids     *mempoolIDs
+}
```
Comment (evan-forbes) [inline mempool/reactor.go:38]: wait a sec, we don't want the reactor in here, right?

**Review by evan-forbes — COMMENTED** [review]
just had a question about the reactor, the rest LGTM! :+1: :+1:

File: mempool/reactor.go (hunk 1 above)
Comment (tzdybal) [reply mempool/reactor.go:38]: `git add mempool` was not a good way of doing this :D good catch!

**Review by evan-forbes — APPROVED** [review]

**Pull request merged**